### PR TITLE
Add doc for Stable Diffusion on Habana Gaudi

### PR DESCRIPTION
**Pull request by regisss**
This PR adds a page in the documentation that briefly shows how to use Stable Diffusion on Habana Gaudi.

### CODE REVIEW

**Comment by regisss** [comment]
Optimum Habana 1.3.0 just got released so this PR can be merged right after approval.

**Review by patrickvonplaten — APPROVED** [review]
Cool!

**Pull request merged**
